app/controllers/metadata/MetadataController.scala

```diff
@@ -84,13 +84,27 @@ class MetadataController @Inject()(
   private def sendEmailIfVerified(notification: Notification)(implicit hc: HeaderCarrier): Future[Boolean] = {
```

Comment:
The Statement Coverage of the Repo will be dropped if we merge this code, it's still above the pass mark but we want to keep it as high as we can to prevent it slipping drop. I think the coverage of the new case exc/case none is not coverage and that's why it's been reduced. It might be we can't fix this now and that's fine, just ignore this comment then.

[info] All done. Coverage was stmt=[92.29%] branch=[93.90%] (MMCA-4297)
[info] All done. Coverage was stmt=[92.50%] branch=[93.90%] (Main)

Reply:
Have added test case and the stmt coverage is 92.39% as opposed to 92.50 % on main. This is a drop of 0.11 %.